example-project/src/main.ts

```diff
@@ -6,19 +6,24 @@ import dummyPluginDef, {
 async function main() {
   // load plugin config
   const pluginConfig = (await relayerEngine.loadFileAndParseToObject(
-    `./plugins/dummy_plugin/config/${relayerEngine.EnvType.DEVNET.toLowerCase()}.json`
+    `./plugins/dummy_plugin/config/${relayerEngine.EnvType.DEVNET.toLowerCase()}.json`,
   )) as DummyPluginConfig;
 
+  const mode =
+    (process.env.MODE?.toUpperCase() as relayerEngine.Mode) ||
```

Comment:
maybe env variables should be prefixed with `RELAYER_ENGINE`? to avoid collision. I actually have a `MODE` env variable that I use for something else in my local environment 😅